pages/design/rule-design.md

```diff
@@ -150,3 +150,26 @@ The description should:
 > _For example: "This page has a `title` element with content."_
 
 For more details, see [ACT Rules Format: Test Cases](https://www.w3.org/TR/act-rules-format/#test-cases).
+
+## Listed conditions
+
+Applicability and expectations, often have multiple conditions that need to be met for something to be applicable / passed. When there are more than three conditions, listing them all in the running text can be difficult to read and understand. Instead, use a bullet-point list following this format:
+
+> Some condition... for which (all | one of) the following is true:
+> 
+> * (**label 1**:) the (X) has condition A; (and | or)
+> * (**label 2**:) the (X) has condition B; (and | or)
+> * (**label 3**:) the (X) has condition C.
+
+This phrasing is designed to be easily readable, but may not work in every situation. In all cases prefer readability over prescriptive formatting. Additionally to this, keep the following in mind when listing conditions:
+
+- The phrasing "for which (all | one of) the following is" can be modified based on use, such as "to achieve one of the following objectives".
+- Labels are optional, not recommended for lists of 2, and recommended for lists of 4 or more conditions. Keep the labels short, 1 or 2 words. If the label does not help clarify the list, don't use them even for longer lists (for example when the condition is short and the label would just repeat it).
```

Comment:
This raised a question for me: is it okay to have labels on only a subset of the list's items?